README.md

```diff
@@ -53,18 +53,15 @@ chmod +x minio
 ```
 
 ### Snap
-You can install the latest `minio` [snap](https://snapcraft.io), and help testing the most recent changes of the master branch in [all the supported Linux distros](https://snapcraft.io/docs/core/install) with:
-
+You can install the latest `minio` [snap](https://snapcraft.io).
 ```sh
 sudo snap install minio --edge
 ```
 
-Every time a new version of `minio` is pushed to the store, you will get it updated automatically.
-
-You will need to allow the minio snap to observe mounts in the system:
-
+Start `minio` using [snap](https://snapcraft.io).
```

Comment:
No need to link to snap again here. We can just say 

`Start minio using Snap run command`